### PR TITLE
join queue response now contains geolocation IPs

### PR DESCRIPTION
**Pull request by PeterAlanAnderson**

### CODE REVIEW

File: src/matchmaker/matchmaker.js [hunk 1]
```diff
@@ -6,10 +6,21 @@ const geolocationIps = require('../common/constants/constants');
...
+      notLocated: {},
+      NW: {}, // North America West
+      NE: {}, // North America East
+      SA: {}, // Sourth America
+      EU: {}, // Europe
+      ME: {}, // Middle East (Israel)
+      JP: {}, // Japan
+      AU: {}, // Australia
+    };
+    this.queue;
```
Comment (PeterAlanAnderson) [inline src/matchmaker/matchmaker.js:19]: Region specific queues

File: src/matchmaker/matchmaker.js [hunk 2]
```diff
@@ -19,10 +30,11 @@ class Matchmaker {
     // TODO - check if IP already in a queue
     const matcherId = this.createMatcherId();
     const newMatcher = new Matcher(matcherId, req.ip, this.deleteMatcher);
-    this.queue.matcherID = newMatcher;
+    this.queue.notLocated.matcherID = newMatcher;
```
Comment (PeterAlanAnderson) [inline src/matchmaker/matchmaker.js:33]: users go to the "notLocated" queue on creation before doing ping test

File: src/matchmaker/matchmaker.js [hunk 3]
```diff
@@ -66,10 +78,6 @@ class Matchmaker {
     };
     res.json(respObj);
   }
-
-  deleteSelf(matcherId) {
-    console.log('MATCHMAKER DELETED MATCHER ', matcherId);
-  }
```
Comment (PeterAlanAnderson) [inline src/matchmaker/matchmaker.js:72]: Duplicate function - covered by deleteMatcher

File: src/common/constants/constants.js [hunk 4]
```diff
@@ -1,30 +1,30 @@
-exports.geoloactionIps = [
```
Comment (PeterAlanAnderson) [inline src/common/constants/constants.js:1]: Fixed a typo :P

**Review by TomOverland — APPROVED** [review]

**Pull request merged**